apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx

```diff
@@ -47,6 +58,40 @@ export default class WorkshopEnrollmentSchoolInfo extends React.Component {
     this.props.onDelete(pendingDeleteId);
   }
 
+  handleScholarshipStatusChange(enrollment, selection) {
+    this.setState({
+      pendingScholarshipUpdates: [
+        ...this.state.pendingScholarshipUpdates,
+        enrollment.id
+      ]
+    });
+    $.ajax({
+      method: 'POST',
+      url: `/api/v1/pd/enrollment/${enrollment.id}/scholarship_info`,
+      contentType: 'application/json;charset=UTF-8',
+      data: JSON.stringify({scholarship_status: selection.value})
+    }).done(data => {
+      this.setState(state => {
+        const enrollments = state.enrollments.map(enrollment => {
```

Comment:
Is it worth a comment explaining what we're doing in this?  It appears we're building set of enrollments that merge in updated information from the server over existing information?